modules/sfp_leakix.py

```diff
@@ -158,11 +161,18 @@ def handleEvent(self, event):
 
             if services:
                 for service in services:
+                    ip = service.get('ip')
+                    if ip and eventName != "IP_ADDRESS":
+                        evt = SpiderFootEvent("IP_ADDRESS", ip, self.__name__, event)
+                        self.notifyListeners(evt)
                     port = service.get('port')
                     if port:
                         evt = SpiderFootEvent("TCP_PORT_OPEN", eventData + ':' + port, self.__name__, event)
                         self.notifyListeners(evt)
-
+                    hostname = service.get('hostname')
+                    if hostname and eventName == "DOMAIN_NAME" and hostname.endswith(eventData):
+                        evt = SpiderFootEvent("INTERNET_NAME", hostname, self.__name__, event)
```

Comment:
Ideally you should include the same `verify` logic as exists in other modules, and report a `INTERNET_NAME_UNRESOLVED` event if the host doesn't resolve.